### ISSUE TITLE
Accessibility - tabs

### ISSUE THREAD

**Issue by KyleCMHRA** [MHRA/products#93]
**Describe the bug**
When using keyboard-only to navigate the site, there are a few issues:

-On homepage, tabbing skips over the descriptive text
-On the homepage, this also results in the 'skipping' of all the descriptive text apart from one in-text link
-On a results page, tabbing does not at any point indicate the type of listing (SPC, PIL or PAR)

As screen-readers are generally used in conjunction with keyboard, the above info is not at all being read by screen-reader software. This means those users are missing out on vital info.

**To Reproduce**

1. Download Chrome plugin ChromeVox Classic Extension
2. Go to the site
3. Enable the plugin and use Chrome to navigate the homepage, and a search results page
4. See errors

**Expected behavior**
That using the keyboard would result in all the information being highlighted in a logical order, and therefore read out in that order using screen reader software

**Screenshots**

**Additional context**

**Comment by pataruco** [comment]
Hi @KyleCMHRA tab navigation is completely different from Assistive technologies navigation.

All Assistive technologies to browse the Web use the same interface (API).

In this case, ChromeVox Classic which enables VoiceOver need to set a Voice Over Activation keys (called VO keys). This setup pattern is standard on all assistive technologies.

- Please, open  ChromeVox Classic settings
<img width="1920" alt="Screenshot 2019-12-12 at 14 20 08" src="https://user-images.githubusercontent.com/10330222/70722373-18f9b180-1cef-11ea-87e5-31c9d2201668.png">

and set the activation key that you want (the convention is `ctrl` + `alt`) and this will come now **ChromeVox** key
- Open the page again and use now your **ChromeVox** key and the arrows to navigate to the page


Here is a video of myself navigating the page using ChromeVox[voice-over-utility.mov.zip](https://github.com/MHRA/products/files/3956329/voice-over-utility.mov.zip)

I am closing this issue.

Please, let me know if you need anything else
